webknossos-datastore/app/com/scalableminds/webknossos/datastore/jzarr/ZarrHeader.scala

```diff
@@ -50,6 +53,30 @@ case class ZarrHeader(
 object ZarrHeader {
   val FILENAME_DOT_ZARRAY = ".zarray"
 
+  def fromLayer(dataLayer: DataLayer, mag: Vec3Int): ZarrHeader = {
```

Comment:
please add explaining comment that this is used for exposing webknossos layers as zarr via the api (otherwise we may get confused in the future thinking it is about zarr layers on disk)